From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Thu, 12 Dec 2019 10:57:32 -0800
Subject: ANDROID: README.md: Fix indented bullet

Some renderers include this bullet in the code
block. Make sure it is not in the code block
and is still indented.

Change-Id: Ic7fd075eec2318934dac7b2f75a8f0fd75304950
Signed-off-by: Todd Kjos <tkjos@google.com>
---
 README.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index 1797579c7de3..110e8535b111 100644
--- a/README.md
+++ b/README.md
@@ -36,7 +36,7 @@ Additional requirements are listed below based on patch type
 
         Signed-off-by: Fred Jones <fred.jones@foo.org>
 ```
-        - then Joe Smith would upload the patch for the common kernel as
+>- then Joe Smith would upload the patch for the common kernel as
 ```
         UPSTREAM: important patch from upstream
 
@@ -86,7 +86,7 @@ must be a stable maintainer branch (not rebased, so don't use `linux-next` for e
 
         Signed-off-by: Fred Jones <fred.jones@foo.org>
 ```
-        - then Joe Smith would upload the patch for the common kernel as
+>- then Joe Smith would upload the patch for the common kernel as
 ```
         FROMGIT: important patch from upstream
 
